common/src/main/java/com/microsoft/identity/common/internal/platform/AndroidDeviceKeyManager.java

```diff
@@ -22,170 +22,57 @@
 
-    private static final String TAG = DeviceKeyManager.class.getSimpleName();
-    private final KeyStore mKeyStore;
-    @Getter
-    private final String mKeyAlias;
-    private final Supplier<byte[]> mThumbprintSupplier;
+    private static final String TAG = AndroidDeviceKeyManager.class.getSimpleName();
 
     @Builder
-    public DeviceKeyManager(@NonNull final KeyStore keyStore, @NonNull final String keyAlias,
-                            @NonNull final Supplier<byte[]> thumbprintSupplier) throws KeyStoreException {
-        this.mKeyAlias = keyAlias;
-        this.mThumbprintSupplier = thumbprintSupplier;
-        this.mKeyStore = keyStore;
-    }
-
-
-    @Override
-    public boolean exists() {
-        boolean exists = false;
-
-        try {
-            exists = mKeyStore.containsAlias(mKeyAlias);
-        } catch (final KeyStoreException e) {
-            Logger.error(
-                    TAG,
-                    "Error while querying KeyStore",
-                    e
-            );
-        }
-
-        return exists;
-
-    }
-
-    @Override
-    public boolean hasThumbprint(@NonNull final byte[] thumbprint) {
-        return Arrays.equals(thumbprint, mThumbprintSupplier.get());
-    }
-
-    @Override
-    public Date getCreationDate() throws ClientException {
-        try {
-            return mKeyStore.getCreationDate(mKeyAlias);
-        } catch (final KeyStoreException e) {
-            Logger.error(
-                    TAG,
-                    "Error while getting creation date for alias " + mKeyAlias,
-                    e
-            );
-            throw new ClientException(ClientException.KEYSTORE_NOT_INITIALIZED, e.getMessage(), e);
-        }
-    }
-
-    @Override
-    public boolean clear() {
-        boolean deleted = false;
-
-        try {
-            mKeyStore.deleteEntry(mKeyAlias);
-            deleted = true;
-        } catch (final KeyStoreException e) {
-            Logger.error(
-                    TAG,
-                    "Error while clearing KeyStore",
-                    e
-            );
-        }
-
-        return deleted;
-    }
-
-
-    /**
-     * Retrieve the entry stored in this particular alias.
-     *
-     * @return the Entry in question, or null if the entry does not exist.
-     * @throws UnrecoverableEntryException
-     * @throws NoSuchAlgorithmException
-     * @throws KeyStoreException
-     */
-    @SuppressWarnings("unchecked")
-    @Override
-    public K getEntry() throws UnrecoverableEntryException, NoSuchAlgorithmException, KeyStoreException {
-        return (K) mKeyStore.getEntry(mKeyAlias, null);
+    public AndroidDeviceKeyManager(@NonNull final KeyStore keyStore, @NonNull final String keyAlias) throws KeyStoreException {
+        super(keyStore, keyAlias, null);
     }
 
     @Override
-    public void importKey(@NonNull final byte[] jwk, @NonNull final String algorithm) throws ClientException {
```

Comment:
The way Github is rendering these changes is strange. This method isn't renamed, it actually just move to the parent abstract class.